### PR TITLE
[WebProfilerBundle] Cleanup profiler leftover

### PR DESCRIPTION
**Pull request by ro0NL**
| Q             | A
| ------------- | ---
| Branch?       | 3.2
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #... 
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

Leftover from 2.8

### CODE REVIEW

File: src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php [hunk 1]
```diff
@@ -108,7 +108,7 @@ public function panelAction(Request $request, $token)
         }
 
         if (!$profile = $this->profiler->loadProfile($token)) {
-            return new Response($this->twig->render('@WebProfiler/Profiler/info.html.twig', array('about' => 'no_token', 'token' => $token, 'request' => $request)), 200, array('Content-Type' => 'text/html'));
+            throw new NotFoundHttpException(sprintf('Token "%s" was not found in the database.', $token));
```
Comment (ro0NL) [inline src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php]: This changes the behavior a little bit, as you break out of the profiler layout for an unknown token (which seems consistent with an unknown ?panel= param btw.)

I think it's negligible :)

**Comment by robfrawley** [comment]
:-1: As you mentioned, this takes the user outside of the profiler environment. Most importantly, this removes the relevant links in the left menu that provide quick access to "Last 10", "Latest", "Search", etc. These allow you to quickly get back to a working profile, instead of having to manually remove the token from the URL in the browser window.

Previous behavior:
symfony_token-not-found-error

PR behavior:
symfony_token-not-found-exception

I'll often have a profile open that I've been looking at while going back and forth between some code, and then at some point it will have been cleaned up and removed, so when I reload the page or navigate elsewhere in the profiler it won't exist any longer, and I *regularly utilize the left navigation to get back to a sane page*.

**Comment by ro0NL** [comment]
Fixed then :+1:

**Review by javiereguiluz — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @ro0NL.

**Pull request merged**